src/main/java/org/generationcp/commons/util/FileNameGenerator.java

```diff
@@ -8,58 +8,30 @@
 public class FileNameGenerator {
 	private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyyMMdd");
 	private static final SimpleDateFormat TIME_FORMAT = new SimpleDateFormat("hhmmss");
-	private static final int MAX_SIZE = 256;
-	private static final int MAX_SIZE_NAME = 200;
+	public static final int MAX_SIZE = 100;
+	public static final int MAX_SIZE_WO_EXTENSION = 96;
+
+	/**
+	 *
+	 * @param origFinalName
+	 * @param fileExt
+	 * @return Generated Filename filename_username_date_time_.fileextension
+	 */
+	public static String generateFileName(final String origFinalName, final String fileExt) {
```

Comment:
Overload generateFileName so no need to have null parameter values in calls with no file extension:
`public static String generateFileName(final String origFinalName) {
		return generateFileName(origFinalName, "");
	}`